source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst

```diff
@@ -752,11 +802,11 @@ Assign Hostname
       alphanumeric character but may also contain hyphens. Maximum length of
       63 characters.
 
-   .. figure:: figures/bare-metal-install-beta-26.png
+   .. figure:: figures/bare-metal-install-beta-27.png
```

Comment:
For some reason, the hostname screenshot is appearing much larger than all other screenshots when I use preview viewing.

Reply:
Revised. Thanks.